doc/classes/Object.xml

```diff
@@ -693,6 +693,7 @@
 			<return type="int" />
 			<description>
 				Returns the object's unique instance ID. This ID can be saved in [EncodedObjectAsID], and can be used to retrieve this object instance with [method @GlobalScope.instance_from_id].
+				[b]Warning:[/b] This ID is specific to the running session. This means that it won't match the same object if sent over a network, or loaded from a file at a later time.
```

Comment:
Maybe saying "useful" really hammers it. It's what the outside reader is most 
Equality is very tricky to explain here. `A == B` is true only if both point to the same object, **not** if all of its values and properties are the exact same (which is where the humane confusion stems from).
```suggestion
				[b]Note:[/b] The returned ID is only useful for the current session. It won't correspond to a similar object if the ID is sent over a network, or loaded from a file at a later time.
```

Reply:
Good idea, will work on something!